js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorerSidebar.tsx

```diff
@@ -574,3 +745,17 @@ const GrayOnHoverBox = styled(Box)`
     visibility: hidden;
   }
 `;
+
+const ButtonGroupWrapper = styled.div`
+  > * {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    > * {
+      place-content: center;
+    }
+  }
+`;
+
+function nodeId(node: TreeNodeType | FolderNodeType) {
+  return 'path' in node ? node.path : node.id;
+}
```

Comment:
It's super confusing that the nodes with a path HAVE an id, but the nodeId is the path and not the id? If we're going to do this, I wonder if we could remove the `id` on the node entirely so that you're forced to use this helper. It's v scary that if I write `node.id` and `nodeId(node)` I get different results for an asset node.